gostream/ffmpeg/avutil/avutil.go

```diff
@@ -73,6 +73,44 @@ func ptr(buf []byte) *C.uint8_t {
 	return (*C.uint8_t)(unsafe.Pointer(h.Data))
 }
 
+// SetFrameFromImgMacroAlign sets the frame from the given image.YCbCr
+// Adding line padding to the image to ensure that the data is aligned
+// to the given boundary
+func (f *Frame) SetFrameFromImgMacroAlign(img *image.YCbCr, boundary int) {
+	// Calculating padded strides
+	// Rounding up to next multiple of boundary value
+	paddedYStride := ((img.YStride + boundary - 1) / boundary) * boundary
+	// UV half the Y stride for 4:2:0
+	paddedCbCrStride := paddedYStride / 2
+
+	// Allocate new buffers with padding
+	// These will be freed by the GC
+	paddedY := make([]byte, paddedYStride*img.Rect.Dy())
```

Comment:
Not entirely happy about having to asign new memory here and copy over and would be open to a better solution.  I do not see a noticeable performance hit in the overall stream.

Reply:
Good call out. I think in the grand scheme of things, this isn't too bad, considering 99% of the time, we're serving images every 16 to 33 ms for 60 and 30 fps, respectively.